components/SectionHeadline.js

```diff
@@ -0,0 +1,9 @@
+import styled, { css } from "styled-components";
+
+export default function SectionHeadline({ headline }) {
+  return <StyledSectionHeadline>{headline}</StyledSectionHeadline>;
+}
+
+const StyledSectionHeadline = styled.h2`
+  margin-bottom: 0.5rem;
+`;
```

Comment:
Here its the same for the headline (but you must write the headline text between the Component tags like this: `<SectionHeadline>This is the text</SectionHeadline>`